test/utils/FakeObjects.scala

```diff
@@ -57,4 +59,112 @@ trait FakeObjects {
     uploadedFiles = Seq(evidence),
     submissionSource = "DFE_UI"
   )
+
+
+  val createValidRequest: GroupAccountSubmission = GroupAccountSubmission(
+    id = "acc123",
+    companyName = "Real news Inc",
+    addressId = 9876543L,
+    email = "thewhitehouse@potus.com",
+    phone = "01987654",
+    isAgent = false,
+    individualAccountSubmission = IndividualAccountSubmissionForOrganisation(
+      externalId = "Ext123",
+      trustId = "trust234",
+      details = IndividualDetails(
+        firstName = "Donald",
+        lastName = "Trump",
+        email = "therealdonald@potus.com",
+        phone1 = "123456789",
+        phone2 = Some("987654321"),
+        addressId = 24680L
+      )
+    )
+  )
+
+  val expectedCreateValidResponse =
+    GroupId(id = 654321L, message = "valid group id", responseTime = 45678)
+
+  val expectedGetValidResponse1 = Some(
+    GroupAccount(
+      id = 2,
+      groupId = "gggId",
+      companyName = "Fake News Inc",
+      addressId = 345,
+      email = "therealdonald@potus.com",
+      phone = "9876541",
+      isAgent = false,
+      agentCode = None))
+
+  val expectedAgentOrganisation = AgentOrganisation(
```

Comment:
lets not call this expected, the fake objects should have no criteria in the name around expected of initial, it should be optionalGroup account, which explains what object you are using when you call this.

Reply:
i agree...I moved them to FakeObjects and forgot to change the names